### PR TITLE
Adds calm book theme

### PR DESCRIPTION
**Pull request by LukeHalasy**

### CODE REVIEW

File: runtime/queries/toml/highlights.scm [hunk 1]
```diff
@@ -7,6 +7,12 @@
   (quoted_key)
 ] @type)
 
+(table [
+  (bare_key)
+  (dotted_key)
+  (quoted_key)
+] @key.declaration)
+
```
Comment (pascalkuthe) [inline runtime/queries/toml/highlights.scm:15]: there are theme key changes and even an unrelated (and incorrect) code change (which will cause build failures) bundled with this PR. A PR adding a theme should not contain such unrelated changes, please remove them

File: runtime/queries/rust/highlights.scm [hunk 2]
```diff
@@ -253,10 +253,10 @@
   ])
 
 (function_item
-  name: (identifier) @function)
+  name: (identifier) @function.declaration)
 
 (function_signature_item
-  name: (identifier) @function)
+  name: (identifier) @function.declaration)
```
Comment (pascalkuthe) [inline runtime/queries/rust/highlights.scm:260]: here

File: helix-lsp/src/lib.rs [hunk 3]
```diff
@@ -946,6 +946,7 @@ pub fn find_lsp_workspace(
         current_dir.join(file)
     };
     file = path::get_normalized_path(&file);
+    workspace = true;
```
Comment (pascalkuthe) [inline helix-lsp/src/lib.rs:949]: and here

**Review by pascalkuthe — CHANGES_REQUESTED** [review]

**Comment by lukehalasy-anchorage** [comment]
Hey @pascalkuthe .
My bad on creating this PR. I meant to create a PR within my fork, not to the main repo. Ill close this PR for now (though I do hope to contribute this theme at some point)

**Pull request closed without merging**